docroot/modules/humsci/hs_layouts/hs_layouts.info.yml

```
@@ -6,6 +6,7 @@ version: 8.2.7
 package: 'Humanities & Sciences'
 dependencies:
   - layout_discovery
+  - drupal/dashboard
```

This is not the right format for dependencies. See https://www.drupal.org/docs/develop/theming-drupal/defining-a-theme-with-an-infoyml-file#dependencies

```suggestion
  - dashboard:dashboard
```

---

Oh 👀 thanks!